$NetBSD$

ldt stuff only on linux. it doesn't work well enough on netbsd
and building 32bit.

--- xbmc/cores/DllLoader/CMakeLists.txt.orig	2017-05-24 20:49:32.000000000 +0000
+++ xbmc/cores/DllLoader/CMakeLists.txt
@@ -24,8 +24,8 @@ if(NOT CORE_SYSTEM_NAME STREQUAL windows
                       SoLoader.cpp)
   list(APPEND HEADERS mmap_anon.h
                       SoLoader.h)
-  if(NOT CORE_SYSTEM_NAME STREQUAL freebsd)
-    list(APPEND SOURCES ldt_keeper.c) 
+  if(CORE_SYSTEM_NAME STREQUAL linux)
+    list(APPEND SOURCES ldt_keeper.c)
     list(APPEND HEADERS ldt_keeper.h)
   endif()
 else()
